packages/gatsby-cli/src/init-starter.ts

```diff
@@ -1,33 +1,34 @@
-/* @flow */
-const { execSync } = require(`child_process`)
-const execa = require(`execa`)
-const hostedGitInfo = require(`hosted-git-info`)
-const fs = require(`fs-extra`)
-const sysPath = require(`path`)
-const report = require(`./reporter`)
-const url = require(`url`)
-const isValid = require(`is-valid-path`)
-const existsSync = require(`fs-exists-cached`).sync
-const { trackCli, trackError } = require(`gatsby-telemetry`)
-const prompts = require(`prompts`)
-const opn = require(`better-opn`)
-
-const {
-  getPackageManager,
-  promptPackageManager,
-} = require(`./util/configstore`)
-const isTTY = require(`./util/is-tty`)
-const spawn = (cmd: string, options: any) => {
+import opn from "better-opn"
+import { execSync } from "child_process"
+import execa from "execa"
+import { sync as existsSync } from "fs-exists-cached"
+import fs from "fs-extra"
+import { trackCli, trackError } from "gatsby-telemetry"
+import hostedGitInfo from "hosted-git-info"
+import isValid from "is-valid-path"
+import sysPath from "path"
+import prompts from "prompts"
+import url from "url"
+
+import report from "./reporter"
+import { getPackageManager, promptPackageManager } from "./util/configstore"
+import isTTY from "./util/is-tty"
+
+const spawnWithArgs = (
+  file: string,
+  args: string[],
+  options?: any
```

Comment:
is it possible to type this? seems like it should be available maybe on `execa` or something?

Reply:
I'm going to merge because this is the only concern i see, but would love to see this get fixed in a follow up!